code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -614,24 +614,33 @@ emp_act
 
 			playsound(loc, 'sound/weapons/slice.ogg', 25, TRUE, -1)
 			visible_message("<span class='danger'>[M] has slashed at [src]!</span>", \
- 				"<span class='userdanger'>[M] has slashed at [src]!</span>")
+				"<span class='userdanger'>[M] has slashed at [src]!</span>")
 
 			apply_damage(damage, BRUTE, affecting, armor_block)
 			add_attack_logs(M, src, "Alien attacked")
 			updatehealth("alien attack")
+			#define ARM_MULTI 1.15
```

Comment:
this name isn't very descriptive. maybe `XENO_ARMOR_MOD` would be a better name?

You will also want to add an `#undef DEFINE_NAME_HERE` below this proc, because this define isn't used anywhere else.

Reply:
There is an #undef below, but as mentioned below I will move this to before and after the proc for clarity when looking through the code.